include/wx/gtk/window.h

```diff
@@ -207,6 +207,8 @@ class WXDLLIMPEXP_CORE wxWindowGTK : public wxWindowBase
     virtual void GTKHandleRealized();
     void GTKHandleUnrealize();
 
+    void GTKEnableOverlay();
```

Comment:
Should have a comment explaining that it does nothing if overlay can't be enabled. Maybe even rename it to `GTKEnableOverlayIfPossible()`. OTOH it would be even better to avoid calling it for all windows as discussed above.

Reply:
I changed it's implementation and now it returns `false` on failure.